features/account/verification/email_verification.feature

```diff
@@ -0,0 +1,53 @@
+@email_verification
+Feature: Verifying account's email address
+    In order to take full advantage of my account
+    As a User
+    I want to be able to verify my email address
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And there is a "valkyrie@cain.com" user
+        And this user is not verified
+
+    @ui
+    Scenario: Getting verified after clicking the link in the verification message
+        Given a verification email has already been sent to "valkyrie@cain.com"
+        When I use it to verify
+        Then I should be notified that the verification was successful
+        And my account should be verified
+
+    @ui
+    Scenario: Being unable to verify with invalid token
+        When I try to verify using email "valkyrie@cain.com" and token "twinklelittlestar"
```

Comment:
It is a @ui scenario, so from the ui point of view... how do you verify using token? is it clear enough?